src/service.rs

```diff
@@ -47,6 +47,11 @@ where
     ///  - `name`=`<name>` a services's name
     #[serde(serialize_with = "crate::docker::serialize_as_json")]
     pub filters: HashMap<T, Vec<T>>,
+
+		/// Include service status, with count of running and desired tasks.
+    #[serde(serialize_with = "crate::docker::serialize_as_json")]
+    #[serde(default)]
```

Comment:
I guess this is not needed either ? It's really only useful if you're deserializing.. or am I wrong about that?